articles/install-guide/csinstall.md

```diff
@@ -0,0 +1,142 @@
+---
+title: Develop with Q# + C#
+author: natke
+ms.author: nakersha
+ms.date: 9/30/2019
+ms.topic: article
+ms.custom: how-to
+uid: microsoft.quantum.install.cs
+---
+# Develop with Q# + C#
+
+Install the QDK to develop C# host programs to call Q# operations.
+
+Q# is a member of the .NET ecosystem and therefore is built to play well with other .NET languages--specifically C#. You can work with this pairing inside different development environments:
+
+- Q# + C# using Visual Studio (Windows)
+- Q# + C# using Visual Studio Code (Windows, Linux and Mac)
+- Q# + C# using the `dotnet` command-line tool
+
+## Develop with Q# + C# using Visual Studio
+
+Visual Studio offers a rich environment for developing Q# programs. The Q# Visual Studio extension contains templates for Q# files and projects as well as syntax highlighting, code completion and IntelliSense support.
+
+
+1. Pre-requisites
+
+    - [Visual Studio](https://visualstudio.microsoft.com/downloads/) 16.3, with the .NET Core cross-platform development workload enabled
+
+1. Install the Q# Visual Studio extension
+
+    - Download the [Visual Studio extension](https://marketplace.visualstudio.com/items?itemName=quantum.DevKit)
+    - Add the extension to Visual Studio
+
+1. Verify the installation by creating a `Hello World` application
+
+    - Create a new Q# application
+
+        - Go to **File** -> **New** -> **Project**
+        - Type `Q#` in the search box
+        - Select **Q# Application**
+        - Select **Next**
+        - Choose a name and location for your application
+        - Select **Create**
+
+    - Inspect the project
+
+        You should see that two files have been created: `Driver.cs`, which is the C# host application; and `Operation.qs`, which is a Q# program that defines a simple operation to print a message to the console.
+
+    - Run the application
+
+        - Select **Debug** -> **Start Without Debugging**
+        - You should see the text `Hello quantum world!` printed to a console window.
+
+> [!NOTE]
+> * If you have multiple projects within one Visual Studio solution, all projects contained in the solution need to be in the same folder as the solution, or in one of its subfolders.  
+
+## Develop with Q# + C# using Visual Studio Code
+
+Visual Studio Code (VS Code) offers a rich environment for developing Q# programs on Windows, Linux and Mac.  The Q# VS Code extension includes support for Q# syntax highlighting, code completion, and Q# code snippets.
+
+1. Pre-requisites
+
+   - [VS Code](https://code.visualstudio.com/download)
+   - [.NET Core SDK 3.0 or later](https://www.microsoft.com/net/download)
```

Comment:
I believe we will be switching to 3.1 as of the next release. We need to remember to update this.